internal/operator/cluster/upgrade.go

```diff
@@ -27,6 +27,7 @@ import (
 	"github.com/crunchydata/postgres-operator/internal/config"
 	"github.com/crunchydata/postgres-operator/internal/kubeapi"
 	"github.com/crunchydata/postgres-operator/internal/operator"
+	cfg "github.com/crunchydata/postgres-operator/internal/operator/config"
```

Comment:
I think the alias we've used in other files is `pgoconfig` that not sure if that replies here.

That said, I would big a more descript alias, given we have `config` above.

Reply:
I didn't see this specific package with that alias, but I'm happy to make it more descriptive.
I updated it to `pgoconfig`, as suggested.